docs/develop/teams-toolkit-overview.md

```diff
@@ -1,47 +1,90 @@
+    :::image type="content" source="../images/teams-toolkit-create-office-task-pane-capability.png" alt-text="The three options in the App Features Using an Office Add-in dropdown menu. The first option 'Taskpane' is selected.":::
 
-1. In the **Workspace folder** dialog that opens, select the folder where you want to create the project.
+1. In the **Programming Language** dropdown menu, select either **TypeScript** or **JavaScript**.
+1. In the **Framework** dropdown menu, select **Default** or **React**. 
+1. In the **Workspace Folder** dialog that opens, select the folder where you want to create the project.
 1. Give a name to the project (with no spaces) when prompted. Teams Toolkit will create the project with basic files and scaffolding. It will then open the project *in a second Visual Studio Code window*. Close the original Visual Studio Code window.
+
+   > [!NOTE]
+   > The project that's generated is configured to be installable on Excel, Outlook, PowerPoint, and Word. You can edit the manifest and source files as needed to change which Office applications are supported.
+
 1. In the Visual Studio Code **TERMINAL** navigate to the root of the project and run `npm install`.
-1. Before you make changes to the project, verify that you can sideload your Outlook add-in from Visual Studio Code. Use the following steps:
-    1. Ensure that your account in your Microsoft 365 developer tenancy is also an email account in desktop Outlook. If it isn't, follow the guidance in [Add an email account to Outlook](https://support.microsoft.com/office/add-an-email-account-to-outlook-e9da47c4-9b89-4b49-b945-a204aeea6726).
-    1. **Close Outlook desktop**.
-    1. In Visual Studio Code, open Teams Toolkit.
-    1. In the **ACCOUNTS** section, verify that you're signed into Microsoft 365.
-    1. Select **View** | **Run** in Visual Studio Code. In the **RUN AND DEBUG** drop down menu, select the option, **Outlook Desktop (Edge Chromium)**, and then press F5. The project builds and a Node dev-server window opens. This process may take a couple of minutes. Eventually, Outlook desktop will open.
-    1. Open the **Inbox** *of your Microsoft 365 account identity* and open any message. A **Contoso Add-in** tab with two buttons will appear on the **Home** ribbon (or the **Message** ribbon, if you have opened the message in its own window).
-    1. Click the **Show Taskpane** button and a task pane opens. Click the **Perform an action** button and a small notification appears near the top of the message.
-    1. To stop debugging and uninstall the add-in, select **Run** | **Stop Debugging** in Visual Studio Code.
+1. After the installation completes, verify that you can sideload your add-in from Visual Studio Code. The steps to sideload vary depending on the Office application on which you want to test the add-in.
+
```

Comment:
Should we add a section to introduce the "Import an Existing Office Add-in" feature? This feature helps developers to upgrade an existing xml manifest add-in project to the unified manifest and project structure.

The steps will be:

1. Select **Create a New App**.
1. In the **New Project** dropdown menu, select **Office Add-in**.
1. In the **App Features Using an Office Add-in** dropdown menu, select **Import an Existing Office Add-in**. 
1. In the **Existing add-in project folder** dialog that opens, select the folder of your existing add-in project.
1. In the **Select import project manifest file** dialog that opens, select the manifest of your existing add-in project.
1. In the **Workspace Folder** dialog that opens, select the folder where you want to create the project.
1. Give a name to the project (with no spaces) when prompted. Teams Toolkit will create the project with basic files and scaffolding. It will then open the project *in a second Visual Studio Code window*. Close the original Visual Studio Code window.
![{E7383AAA-B8EC-4D27-8D04-6ED69D4C6A03}](https://github.com/user-attachments/assets/eb96fba0-1596-43f6-a841-e7610c0eeadb)